kubernetes/README.md

```diff
@@ -0,0 +1,17 @@
+![image](../images/confluent-logo-300-2.png)
+
+# Overview
+
+This section of the examples repository will contain a variety of uses cases for deployment of Confluent Platform and supporting applications deployed via [Confluent Operator](https://docs.confluent.io/current/installation/operator/index.html).
+
+# Use Cases
```

Comment:
Instead of `Use Cases`, how about `Running the Demos` (or some other explicit verbiage)?  This will help direct users to click on the right link to run the demo

Reply:
@ybyzek cleaned up some, let me know what you think of the current version.

Reply:
Looks good!